docs/using-fit-on-fhir.md

```diff
@@ -2,18 +2,54 @@
 
 In your resource group, find the Authorization Function. It will be named **authorize-{YOUR_BASE_NAME}**, where, YOUR_BASE_NAME is the basename parameter provided when the resources were deployed.
 ![Auth Function in Resource Group](../media/auth-function-resource-group.png)
 
 In the overview section of the Function copy the base URL. Append the base URL with **api/googlefit/authorize**.
 ![Auth Function Base URL](../media/auth-function-url.png)
+
+## Configuring Access to your Authorization Function
+
+Access to the Authorization function can be configured to either allow for anonymous login, or login with authentication.  This setting is configured by setting
+*AuthenticationConfiguration__IsAnonymousLoginEnabled* to true for anonymous logins, and false for login with authentication.  This can be found in can be found in Settings->Configuration.
+
+**Required Query Parameters**:
+
+Anonymous login requests require two query parameters, *external_id* and *external_system*.  The *external_id* and *external_system* are used to create a FHIR [Identifier](http://hl7.org/fhir/datatypes.html#Identifier) that is stored in the [Patient Resource](http://hl7.org/fhir/patient.html). This Identifier can be used to link the Patient Resource to a user (or patient) in a different system.
+
+_An example anonymous login request might look like: https://authorize-fitonfhir.azurewebsites.net/api/googlefit/authorize?external_id=externalPatientA&external_system=externalSystem&redirect_url=https://www.microsoft.com/_
+
+Login with authentication requests require passing a valid OAuth2 access token, and one query parameter *redirect_url*.  *redirect_url* represents the URL that
+the Authorization function will redirect to, once authorization with Google is complete.  The URL contained in *redirect_url* must match a URL that is on the
+approved list for the Authorization function.  The approved list of redirect URLs can be found in Settings->Configuration and is labeled *AuthenticationConfiguration__RedirectUrls*.
+*external_id* and *external_system* query parameters are not allowed when login with authentication is enabled.  Including either of these in a request will result in
+a Bad Request (400) response.
+
+In addition to declaring the approved list of redirect URLs, it is also necessary to configure the *Audience* and *Identity Providers* that will be used during the authentication process.
+The *Audience*, which is equivalent to the aud claim in the OAuth2 access token, can be declared by setting the value for *AuthenticationConfiguration__Audience* in Settings->Configuration.
+The *Identity Providers*, which are the equivalent to the iss claim in the OAuth2 access token, can be declared by setting the value for *AuthenticationConfiguration__IdentityProviders* in Settings->Configuration.
+*AuthenticationConfiguration__IdentityProviders* is a list, so it is possible to declare multiple providers if necessary, separating each provider by a comma.
```

Comment:
It seems like this belongs in the "Configuring Access to your Authorization Function" section of the document.